src/util/http/websocket/WebSocketSession.cpp

```diff
@@ -78,7 +89,9 @@ net::awaitable<void> WebSocketSession::acceptAndWait(
   } catch (boost::system::system_error& error) {
     // Gracefully end if socket was closed
     if (error.code() == beast::websocket::error::closed ||
-        error.code() == net::error::operation_aborted) {
+        error.code() == net::error::operation_aborted ||
+        error.code() == net::error::eof ||
+        error.code() == net::error::connection_reset) {
```

Comment:
For that many checks consider using e.g. `ad_utility::contains` or something to reduce the repetition of `error.code()` and `||`.